t/mojo/morbo.t

```diff
@@ -71,6 +71,7 @@ app->log->level('fatal');
 Mojo::IOLoop->next_tick(sub { path('$started')->touch });
 
 get '/hello' => {text => 'Hello World!'};
+get '/start' => {text => 'Started Morbo!'};
```

Comment:
There is nothing special about morbo, these tests don't need to exist if the hook was implemented propoerly in `Mojo::Server::Daemon`.

Reply:
Instead what does need tests is prefork.

Reply:
You are right of course.  But should not we always make sure the proper semantics we want is indeed covered through all the different high-level use cases?
Sure, morbo relies on Mojo::Server::Daemon today, but what if, one day, this uses a new implementation and we forget about the semantic we want for the hook we are introducing here?

Reply:
No, morbo will always be a full reload of `Mojo::Server::Daemon`.

Reply:
OK, I trust your judgment.  Removing specific morbo testing!